docs/logs/customizing-the-sdk/README.md

```diff
@@ -30,7 +30,53 @@ TODO
 
 ### SetResourceBuilder
 
-TODO
+A `ResourceBuilder` provides a `Resource` to the `OpenTelemetryLoggerProvider`.
+A [Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+is an immutable representation of the entity producing telemetry as
+[Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes).
+
+The snippet below demonstrates creating a default `ResourceBuilder` and adding
+service metadata.
+
+```csharp
+using var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder
+        .AddOpenTelemetry(options =>
+        {
+            options.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
+                serviceName: "MyService",
+                serviceVersion: "1.0.0"
+                ));
+            options.AddConsoleExporter();
+        });
+});
+
+var logger = loggerFactory.CreateLogger<Program>();
+
+logger.LogInformation("Hello Information");
+```
+
+This example generates the following output which includes the customized
+service metadata:
+
+```text
+LogRecord.TraceId:            00000000000000000000000000000000
+LogRecord.SpanId:             0000000000000000
+LogRecord.Timestamp:          0001-01-01T00:00:00.0000000Z
```

Comment:
Where is timestamp coming from? It seems wrong.

Reply:
That was the format for the output, I manually zeroed out the values. 
I contemplated removing most of the example output and only showing the last three lines which show the resource.

Reply:
You can put a valid timestamp like the 1st second of 1970.

Reply:
done. changed to `1970-01-01T00:00:01.0000000Z`